app/storage/encrypted_storage.py

```diff
@@ -1,34 +1,56 @@
-import json
-
 from app import settings
 from app.cryptography.jwe_decryption import JWEDirDecrypter
 from app.cryptography.jwe_encryption import JWEDirEncrypter
-from app.storage.abstract_server_storage import AbstractServerStorage, logger
+from app.storage.abstract_server_storage import AbstractServerStorage
+from app.utilities.strings import to_str
+from app.utilities.strings import to_bytes
+import json
+import hashlib
+import logging
+
+logger = logging.getLogger(__name__)
 
 
 class EncryptedServerStorageDecorator(AbstractServerStorage):
 
     ENCRYPTED_KEY = 'encrypted'
```

Comment:
It's literally just a JSON wrapper round the encrypted data. It just allows me to use the the same classes to store the data whether it's encrypted or not. It was also a bit forward planning, I thought it would be easier to use the Postgres JSON column and/or switch to a NoSQL datastore in the future if the data we're saving is JSON.

It's obviously badly named so I'll change that.